addons/mail/models/mail_channel.py

```diff
@@ -623,7 +623,7 @@ def _check_can_update_message_content(self, message):
 
     def _message_update_content_after_hook(self, message):
         self.ensure_one()
-        self.env['bus.bus']._sendone(self, 'mail.message/insert', {
+        self.env['bus.bus']._sendone(self.uuid, 'mail.message/insert', {
```

Comment:
Don't make that change, keep override in livechat instead. We want to keep using self and not uuid in the base case.

Reply:
okay. Done!!